pcdsdevices/slits.py

```diff
@@ -492,3 +500,49 @@ class PowerSlits(BeckhoffSlits):
 
     rtds = DDCpt(_rtd_fields(RTD, 'rtd', range(1, 9)))
     fsw = Cpt(NotImplementedSignal, ':FSW', kind='normal')
+
```

Comment:
The style check is failing as there are 3 blank lines here instead of 2 (https://pcdshub.github.io/style.html)